app/models/contact_message.rb

```diff
@@ -0,0 +1,21 @@
+class ContactMessage < ActiveRecord::Model
+  attr_accessor :copy, :message, :send_name, :send_email,
+    :email
+
+  def initialize(contact: )
+  end
+
+  def copy
+    if copy.present? && send_email.present?
+      send_email
```

Comment:
Here you are using a verb in an attribute name. I don't like having a send_mail method which does not send any mails.